shared/feature/networkuser/src/main/java/com/epmedu/animeal/networkuser/data/repository/NetworkRepositoryImpl.kt

```diff
@@ -1,19 +1,37 @@
 package com.epmedu.animeal.networkuser.data.repository
 
+import com.epmedu.animeal.auth.AuthAPI
 import com.epmedu.animeal.auth.AuthRequestHandler
 import com.epmedu.animeal.auth.UserAttributesAPI
+import com.epmedu.animeal.common.data.wrapper.ApiResult
+import com.epmedu.animeal.networkuser.data.mapper.AuthUserAttributesToIsPhoneVerifiedMapper
+import com.epmedu.animeal.networkuser.data.mapper.AuthUserAttributesToProfileMapper
 import com.epmedu.animeal.networkuser.data.mapper.ProfileToAuthUserAttributesMapper
 import com.epmedu.animeal.networkuser.domain.repository.NetworkRepository
 import com.epmedu.animeal.profile.data.model.Profile
 import javax.inject.Inject
 
 class NetworkRepositoryImpl @Inject constructor(
-    private val profileToAuthUserMapper: ProfileToAuthUserAttributesMapper,
+    private val authAPI: AuthAPI,
     private val userAttributesAPI: UserAttributesAPI,
+    private val authUserAttributesToIsPhoneVerifiedMapper: AuthUserAttributesToIsPhoneVerifiedMapper,
+    private val authUserAttributesToProfileMapper: AuthUserAttributesToProfileMapper,
+    private val profileToAuthUserMapper: ProfileToAuthUserAttributesMapper,
 ) : NetworkRepository {
 
-    override suspend fun fetchNetworkUserAttributes(authRequestHandler: AuthRequestHandler) {
-        userAttributesAPI.fetchUserAttributes(authRequestHandler)
+    override suspend fun isPhoneNumberVerified(): Boolean {
+        val result = userAttributesAPI.fetchUserAttributes()
+        return result is ApiResult.Success && authUserAttributesToIsPhoneVerifiedMapper.map(result.data)
```

Comment:
It looks a bit confusing for me.
Why does a mapper returns boolean? 
Maybe it's not a mapper after all? :)

Reply:
@ArtemKleinschmidt if I mapped the api result would it be better?

Reply:
Replaced with `isVerified` extension